content/TrueCommand/Alerts/AlertManage.md

```diff
@@ -6,125 +6,179 @@ aliases:
+3. Finished: To create the new alert rule, click **CREATE ALERT**. To start over, click **RESET**.
+
+{{< /tab >}}
+{{< tab "Alert Services">}}
 
 {{< hint info >}}
-Configurable Alert Services are only available for local installations or containerized TrueCommand deployments.
+Configurable alert services are only available for local installations or containerized TrueCommand deployments.
 TrueCommand Cloud instances are preconfigured to use email alerts; PagerDuty is not an option.
 {{< /hint >}}
 
-TrueCommand uses different services to expand how alerts are communicated to individual users or administrators.
-Individual user accounts can use these services to manage how that account is notified of an alert.
-To configure an alert service plugin, open the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and click **Alert Services**.
+TrueCommand uses different service plugins to expand how alerts are communicated to individual users or administrators.
+Individual user accounts can use these services to manage how that account is notified of a system alert.  
+
+## Configuring Alert Services
 
-![Alert Services](/images/TrueCommand/2.0/AlertServices.png "Alert Services")
+To configure an alert service plugin, open the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and click **Alert Services**. There are two services listed:
 
-Each plugin has three options.
+* **PagerDuty** is a plugin to configure a pager to receive an alert.
+* **SMTP Email** is a plugin to configure system and user email services.
 
-<mat-icon _ngcontent-ath-c575="" role="img" fontset="mdi" fonticon="mdi-test-tube" class="mat-icon notranslate mdi mdi-test-tube mat-icon-no-color" aria-hidden="true" ng-reflect-font-set="mdi" ng-reflect-font-icon="mdi-test-tube"></mat-icon> : Sent Test Alert
+![Alert Services Screen](/images/TrueCommand/2.1/AlertServiceScreen.png "Alert Services")
 
-<i class="material-icons" aria-hidden="true" title="Settings">settings</i> : Configure Plugin
+Each plugin has three options:
 
-<mat-icon _ngcontent-ath-c200="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">remove_circle</mat-icon> : Clear Plugin Configuration
+* **Send test email** <mat-icon _ngcontent-ath-c575="" role="img" fontset="mdi" fonticon="mdi-test-tube" class="mat-icon notranslate mdi mdi-test-tube mat-icon-no-color" aria-hidden="true" ng-reflect-font-set="mdi" ng-reflect-font-icon="mdi-test-tube"></mat-icon> 
+* **Configure plugin** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> 
+* **Clear plugin configuration** <mat-icon _ngcontent-ath-c200="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">remove_circle</mat-icon> 
```

Comment:
Is this one working on builds?
```suggestion
* **Clear plugin configuration** <mat-icon _ngcontent-ath-c200="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">remove_circle</mat-icon> 
```

Reply:
This was the original icon link content that worked I just moved it to display after the 'label' for it for consistent parallel construction in the bullet list. 
My local build worked, but the hint box I added after it was messed up so I removed the hint box and it worked without issues.